CMakeLists.txt

```diff
@@ -137,7 +137,7 @@ ENDFOREACH()
 
 option(BYN_ENABLE_LTO "Build with LTO" Off)
 if(BYN_ENABLE_LTO)
-  if(NOT CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+  if(NOT CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
is this needed? docs say it allows regexes, but the right side isn't one?

Reply:
Apple's clang has the compiler ID `AppleClang` which MATCHES clang but isn't STREQUAL. Using MATCHES with bare strings seems pretty common in CMake files (it definitely is in LLVM's)

Reply:
I see, sgtm.